src/ui/ResultActions/ResultActionsMenu.ts

```diff
@@ -113,7 +113,7 @@ export class ResultActionsMenu extends Component {
   }
 
   private bindEvents() {
-    $$(this.parentResult).on('click', () => (this.isOpened ? this.hide() : this.show()));
+    $$(this.parentResult).on('click', () => this.show());
```

Comment:
This causes the build to fail because there's now an unused variable (`this. isOpened `).

Does it now close only on blur of the element ?
Does it now close if you click elsewhere in the interface ?

Reply:
Right, I will delete the variable and resubmit.

When using a mouse:
- The menu only shows when you hover the result.
- When you click on the menu, it stays there.

When using a mobile:
- The menu shows up when you click on the result.
- It disappears when you click elsewhere or on another result.